templates/api/knowledge-panels/contribution/data_quality_tags.tt.json

```
@@ -0,0 +1,68 @@
+{
+    "level": "info",
+    "topics": [
+        "contribution"
+    ],
+    [%# only expand errors and info %]
+    [% IF panel.tags_type == "data_quality_errors" %]
+        "expand_for": "large",
+    [% ELSIF panel.tags_type == "data_quality_info" %]
+        "expand_for": "large",
+    [% ELSE %]
+        "expanded": false,
+    [% END %]
+    [% IF panel.tags_type == "data_quality_errors" %]
+        "evaluation": "bad",
+        [% icon_name = "data-error" %]
+    [% ELSIF panel.tags_type == "data_quality_warnings" %]
+        "evaluation": "average",
+        [% icon_name = "data-warning" %]
+    [% ELSE %]
+        "evaluation": "good",
+        [% icon_name = "data-info" %]
+    [% END %]
+    "title_element": {
+        "title": "[% lang(panel.tags_type _ '_panel_title') %]",
+        "subtitle": "[% lang(panel.tags_type _ '_panel_subtitle') %]",
+        "icon_url": "[% static_subdomain %]/images/icons/dist/[% icon_name %].svg",
+        "icon_color_from_evaluation": true,
+    },
+    "elements": [
+        [% 
+            # we need description_key variable because we can't use ':' in key directly
+            description_key = 'description:en'
+            action_name = "action_$fix_action"
+        %]
+        [% FOREACH pair IN panel.quality_tags.pairs %]
+            [%
+                fix_action = pair.key
+            %]
+            {
+                "element_type": "text",
+                "text_element": {
+                    "html": `[% FILTER collapse %]
+                        <ul>
+                        [% FOREACH action_pair IN pair.value -%]
```

I think action_pair is confusing, here it's not an action, it's a data quality tag.

Instead I would name the first "pair" above "action_pair", and this one should be "tags_pair" I think.

So something like 
```suggestion
                        [% FOREACH tag_pair IN action_pair.value -%]
```

---

Righ, changed.